Licton springs review nextjs - 101 - upload new content in art page

edalemiguel opened the pull request:
**Resolves:** Issue #101

**Description:**
Uploaded the new edition content to the LSR Art page.

- [x] Accessed the shared OneDrive folder for the latest content
- [x] Uploaded and displayed the content on the /art page
- [x] Alphabetically sorted entries by the artist’s last name (with “Garret_Alannah” prioritized on top)
- [x] Verified all titles, file names, and image quality
- [x] Confirmed correct image rendering in both the list view and detail pages

**How to Test:**
1. Pull the latest changes
2. Run npm run dev
3. Visit /art and click on any artwork
4. Verify the following:
- [ ] Items on the main art page are sorted by last name
- [ ] Images and content match what’s in the shared folder
- [ ] Verify all titles, file names, and image quality
- [ ] Layout looks clean and polished

**Expected Result:**
New artwork displays correctly and in proper order
Images and content match what’s in the shared folder
No image overflows the screen and good image quality
Layout looks clean and polished

**Screenshot of Result:**
![localhost_3000_art_Garret_Alannah](https://github.com/user-attachments/assets/64495899-85a3-4709-afd4-221f473580f9)

![localhost_3000_art (2)](https://github.com/user-attachments/assets/acdeb1fb-2e9d-4f79-87fd-ed7e796707e7)

cshimm reviewed (approved):
All the new art is now loading on the Art page. Nice job

Bejarano03 reviewed (changes requested):
I like how you implemented these changes to load the art onto the art section. I was able to copy this locally, but it appears that I am getting an error with the path to the folder name. This is not letting the images show up. Could you check it out and let me know if maybe it's just a typo? 

Screenshots:
![Screenshot_20250608_151552](https://github.com/user-attachments/assets/e4bc9a61-40a9-4d86-a557-07d8737c22d6)


![Screenshot_20250608_151611](https://github.com/user-attachments/assets/106d1e57-394e-47e9-83bd-66c82684bd49)




It appears that the folder name is Arts2025 and not arts2025.

edalemiguel commented:
I updated the arts2025 to Arts2025 and the images are loading now, however I don't show any depreciated feature:
<img width="913" alt="Screenshot 2025-06-09 at 6 40 00 AM" src="https://github.com/user-attachments/assets/7eb506d4-f796-4f1b-a64a-e14518c04a4b" />

Bejarano03 reviewed (approved):
I was able to get this working, thank you for the changes

edalemiguel commented:
Hi everyone, I fixed the lint error and the depreciation error. I also changed the images in art page so it will show the full image but still scaled down. Lastly, also updated the homepage image. So far everything looks good on my end. I hope it works when tested, please let me know otherwise. Thank you.
